types/react-leaflet/react-leaflet-tests.tsx

```diff
@@ -211,30 +243,34 @@ export class DraggableExample extends Component<any, any> {
 
     toggleDraggable = () => {
         this.setState({ draggable: !this.state.draggable });
-    }
+    };
 
     updatePosition = () => {
-        const { lat, lng } = (this.refs['marker'] as MarkerInstance).leafletElement.getLatLng();
+        const {
+            lat,
+            lng,
+        } = (this.refs.marker as Marker<MarkerProps, Leaflet.Marker>).leafletElement.getLatLng();
```

Comment:
I feel `MarkerInstance` was a bit nicer than everyone having to do this big cast to get the element. Can we make this nicer somehow?